tools/kagami/src/main.rs

```diff
@@ -209,7 +210,7 @@ mod docs {
 
     use super::*;
 
-    impl<E: Debug, C: Configurable<Error = E> + Send + Sync + Default> PrintDocs for C {}
+    impl<E: Debug, C: Configurable<Error = E> + Send + Sync + DocsDefault> PrintDocs for C {}
```

Comment:
This seemed to work as a stopgap when we take away the `#[serde(default)]` on config structs, but we need something more sophisticated here. That would be done at a later stage.

Reply:
Create a ticket, reference in `// TODO`.